.eslintrc

```diff
@@ -10,5 +10,7 @@
     "no-multi-spaces": "error",
     "react/jsx-tag-spacing": "warn",
     "react/jsx-wrap-multilines": "error",
+    "react/sort-comp": "warn",
+    "react/require-default-props": "error",
     }
```

Comment:
Still needs more rules, figure out which ones might be useful for you guys, `eslint-config-airbnb` is a good base.

Also here are react rules that you can read through and pick the once you want https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules

(Doesn't need to be in this PR, but the sooner the better)